web/packages/plugins/analysis-platform/src/utils/CRUD.tsx

```diff
@@ -0,0 +1,130 @@
+import { ANALYSIS_PATH, ASSET_PATH, DEFAULT_DATASOURCE_ID } from '../const'
+import { DmssAPI } from '@dmt/common'
+
+/**
+ * Create a new entity
+ *
+ * @param directory Path to store the entity in
+ * @param body The document (entity) to upload
+ * @param token The token to use for authentication
+ * @param files The files to upload
+ */
+const create = (
+  directory: string,
+  body: any,
+  token: string,
+  files: Blob[] = [],
+  updateUncontained: boolean = false
+): Promise<string> => {
+  const dmssAPI = new DmssAPI(token)
+
+  return new Promise((resolve, reject) => {
+    dmssAPI
+      .explorerAddToPath({
+        dataSourceId: DEFAULT_DATASOURCE_ID,
+        updateUncontained: updateUncontained,
+        document: JSON.stringify(body),
+        directory: directory,
+        files: files.filter((item: any) => item !== undefined),
+      })
+      .then((response: any) => {
```

Comment:
I think "response: any" can be substituted by AxiosResponse\<any\> 

this gives a bit more type information

Reply:
We might even know the response type here no?
`AxiosResponse<TAnalysis>`